CHANGELOG.md

```diff
@@ -8141,7 +8158,8 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 
 ## [v1.0.0] - 2018-02-23
 
-[Unreleased]: https://github.com/nasa/cumulus/compare/v19.0.0...HEAD
+[Unreleased]: https://github.com/nasa/cumulus/compare/v19.1.0...HEAD
+[v19.1.0]: https://github.com/nasa/cumulus/compare/v19.0.0...v19.1.0
```

Comment:
Not blocking given the PR, but should we do something like 18.5->19.1 in the future given the migration path doesn't travel through 19.0?

Reply:
The stance I'm taking with v19 is that it exists but is deprecated. So the CL entry for 19.1 has:

This release contains changes listed here as well as changes listed in v19.0.0,
despite v19.0.0 being deprecated. Please review Changelog entries and Migration Notes for
each Cumulus version between your current version and v19.1.0 as normal.

My sense is that it would be more confusing to skip a version here an elsewhere. I could be wrong though.